platform-ajax/src/main/typescript/com/softicar/platform/ajax/engine/Actions.ts

```diff
@@ -0,0 +1,42 @@
+
+interface Action {
+	execute(): void;
+}
+
+// This action represents a server request. When executed, this
```

Comment:
In most of the other source files, we mimic...
```
/**
 * Javadoc syntax
 */
```
We should probably do the same in TS-transcribed code.
Unless this was a deliberate change of style.

Reply:
I have to check what is the recommended type of comments in TypeScript.

Reply:
converted all comments to TypeDoc